.github/workflows/pull_requests_console.yaml

```diff
@@ -0,0 +1,185 @@
+        with:
+          version: v3.11.1
+
+      - name: Set up chart-testing
+        uses: joejulian/chart-testing-action@v2.3.1-1
+        with:
+          version: v3.8.0-1
+
+      # we're excluding console from testing until we have a way to test it with Redpanda
+      - name: Run chart-testing (list-changed)
+        id: list-changed
+        run: |
+          changed=$(ct list-changed --target-branch ${{ github.event.repository.default_branch }} --config .github/ct-console.yaml)
+          echo $changed
+          if [[ -n "$changed" ]]; then
+            echo changed="true" >> "$GITHUB_OUTPUT"
+          fi
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.5.0
+        if: steps.list-changed.outputs.changed == 'true'
+        with:
+          config: .github/kind.yaml
+
+      - name: Check kind config worked
+        if: steps.list-changed.outputs.changed == 'true'
+        run: kubectl get nodes
+
+      - name: Annotate nodes for rack awareness
+        if: steps.list-changed.outputs.changed == 'true'
+        run: .github/annotate_kind_nodes.sh chart-testing
+
+      - name: install cert-manager
+        if: steps.list-changed.outputs.changed == 'true'
+        run: |
+          helm repo add jetstack https://charts.jetstack.io &&
+          helm install cert-manager --namespace cert-manager \
+            --create-namespace --version v1.11.0 jetstack/cert-manager \
+            --set installCRDs=true --wait --wait-for-jobs
+
+      - name: install prometheus-operator
+        if: steps.list-changed.outputs.changed == 'true'
+        run: |
+          helm repo add prometheus-community https://prometheus-community.github.io/helm-charts &&
+          helm install prometheus prometheus-community/kube-prometheus-stack \
+            --namespace prometheus --create-namespace \
+            --set prometheus.prometheusSpec.serviceMonitorSelectorNilUsesHelmValues=false \
+            --set nodeExporter.enabled=false \
+            --set grafana.enabled=false --set kubeStateMetrics.enabled=false --set alertmanager.enabled=false \
+            --wait --wait-for-jobs
+
+      - name: install metallb
+        if: steps.list-changed.outputs.changed == 'true'
+        run: |
+          helm repo add metallb https://metallb.github.io/metallb &&
+          helm install metallb metallb/metallb -n metallb-system \
+            --create-namespace --version 0.13.10 --wait --wait-for-jobs
+
+      - name: apply metallb resources
+        if: steps.list-changed.outputs.changed == 'true'
+        run: |
+          kubectl -n metallb-system apply -f .github/metallb-config.yaml
+
+        # Would like to enable tls but this requires additional work
+      - name: install redpanda
+        if: steps.list-changed.outputs.changed == 'true'
+        run: |
+          helm repo add redpanda https://charts.redpanda.com &&
+          helm install redpanda redpanda/redpanda --namespace redpanda  \
+            --set tls.enabled=false \
```

Comment:
Good Job.

As a follow up I would suggest to create another Redpanda listener that has TLS enabled.